### PR TITLE
Add distributed domain/array creation commcount test

### PR DESCRIPTION
**Pull request by e-kayrakli**
This PR

- modifies `test/performance/array/distCreate.chpl` to enable it for commcount testing
- adds relevant commcount files
- adjusts `CC-GRAPHFILES`

Todo/test:
- [x] confirm plots are generated correctly
- [x] confirm numbers are mapped correctly
- [x] test on chapcs
  - [x] correctness
  - [x] perflabel cc-

### CODE REVIEW

File: test/performance/array/distCreate-arrays-deinit.cc-perf.graph [hunk 1]
```diff
@@ -0,0 +1,5 @@
+perfkeys: arrDeinit-GETS:, arrDeinit-PUTS:, arrDeinit-ONS:, arrDeinit-GETS:, arrDeinit-PUTS:, arrDeinit-ONS:, arrDeinit-GETS:, arrDeinit-PUTS:, arrDeinit-ONS:
+graphkeys: blockArrDeinit-GETS:, blockArrDeinit-PUTS:, blockArrDeinit-ONS:, cyclicArrDeinit-GETS:, cyclicArrDeinit-PUTS:, cyclicArrDeinit-ONS:, blockCycArrDeinit-GETS:, blockCycArrDeinit-PUTS:, blockCycArrDeinit-ONS:
+repeat-files: distCreate-cc-block.dat, distCreate-cc-cyclic.dat, distCreate-cc-blockCyc.dat
+ylabel: Count
+graphtitle: Destroying distributed arrays
```
Comment (ronawho) [inline test/performance/array/distCreate-arrays-deinit.cc-perf.graph]: Graph title is the same for each graph currently

File: test/performance/array/distCreate.cc-numlocales [hunk 2]
```diff
@@ -0,0 +1 @@
+16
```
Comment (ronawho) [inline test/performance/array/distCreate.cc-numlocales]: I think all the other cc tests use 4 locales. I'm not sure if 16 will cause problems, but if 4 will work let's go with that.

File: test/performance/array/distCreate-arrays-deinit.cc-perf.graph (hunk 1 above)
Comment (e-kayrakli) [reply test/performance/array/distCreate-arrays-deinit.cc-perf.graph]: I don't think so?

Comment (ronawho) [reply test/performance/array/distCreate-arrays-deinit.cc-perf.graph]: Yeah, my mistake

**Comment by e-kayrakli** [comment]
With #14118 and this PR merged, #12930 can be closed

**Pull request merged**